R/BUILDING.md

```diff
@@ -0,0 +1,13 @@
+## Building SparkR on Windows
```

Comment:
It was a rhetorical question, Shiv :) I think you'd want to say something about other platforms, if you are talking about windows. Otherwise it can be pretty confusing.